simplefs/simplefs.go

```diff
@@ -111,23 +115,28 @@ type handle struct {
 // make sure the interface is implemented
 var _ keybase1.SimpleFSInterface = (*SimpleFS)(nil)
 
-// NewSimpleFS creates a new SimpleFS instance.
-func NewSimpleFS(config libkbfs.Config) keybase1.SimpleFSInterface {
-	return newSimpleFS(config)
-}
-
-func newSimpleFS(config libkbfs.Config) *SimpleFS {
+func newSimpleFS(g *libkb.GlobalContext, config libkbfs.Config) *SimpleFS {
```

Comment:
Why do we need the `GlobalContext`?

Reply:
`libkb.HTTPSrv` needs it.

Reply:
oh I see. Too bad there isn't an easy way of working around that :(

Reply:
It looks like it's only used for logging. If we care about it enough, perhaps we can change `libkb.HTTPSrv` to accept a `Logger` interface instead.

Reply:
maybe later :)